ibis/expr/api.py

```diff
@@ -3987,8 +3988,24 @@ def _table_union(left, right, distinct=False):
     -------
     union : TableExpr
     """
-    op = ops.Union(left, right, distinct=distinct)
-    return op.to_expr()
+    return ops.Union(left, right, distinct=distinct).to_expr()
+
+
+def _table_intersect(left, right):
```

Comment:
can you type

Reply:
@jreback There's no type hinting anywhere else in this repo, so wouldn't it be weird to have it here?